docs/subnets/how-to-upgrade.md

```diff
@@ -0,0 +1,469 @@
+# How To Upgrade Your Subnet-EVM
```

Comment:
We need to be more specific with the naming here. `How to Upgrade Your Subnet-EVM Precompile Configuration`